### PR TITLE
implement hover style for WebsiteTabs

### PR DESCRIPTION
**Pull request by lovemecomputer**
Closes https://github.com/carbon-design-system/carbon-website/issues/1219

**changes:**
- implement hover style for tabs in a `WebsiteTabs` component

### CODE REVIEW

File: src/components/WebsiteTabs/website-tabs.scss [hunk 1]
```diff
@@ -35,16 +35,22 @@
   color: $text-01;
   line-height: 1.25;
   padding: $spacing-sm $spacing-05;
+  transition: background $transition--base;
```
Comment (tw15egan) [inline src/components/WebsiteTabs/website-tabs.scss]: Should we be pulling in and consuming the motion package? We've switched away from `$transition--base` and have been using the new `$duration` tokens. Background changes should be `fast-01` (I think). @shixiedesign can you confirm?

**Comment by tw15egan** [comment]
Actually, it looks like we're not using the motion package anywhere in the website. @shixiedesign, is there any issue in the backlog to change the motion on the website?

**Comment by lovemecomputer** [comment]
@tw15egan to your point i'd love to talk about this more! i've been noticing a few little details on the website that we could improve the motion on, and i'd like to make sure to be up to date on this

**Comment by tw15egan** [comment]
@lovemecomputer sweet! I'll keep you in the loop. Going to create an issue now

File: src/components/WebsiteTabs/website-tabs.scss (hunk 1 above)
Comment (shixiedesign) [reply src/components/WebsiteTabs/website-tabs.scss]: Yes! We should be using `duration--fast-01` for hover animations. Thanks!

**Comment by tw15egan** [comment]
FYI: https://github.com/carbon-design-system/carbon-website/issues/1564

**Pull request merged**